src/databricks/labs/ucx/hive_metastore/tables.scala

```diff
@@ -36,8 +36,10 @@ def metadataForAllTables(databases: Seq[String], queue: ConcurrentLinkedQueue[Ta
           failures.add(TableError("hive_metastore", databaseName, tableName, s"result is null"))
           None
         } else {
+          val upgrade_to=table.properties.get("upgraded_to")
           Some(TableDetails("hive_metastore", databaseName, tableName, table.tableType.name, table.provider.orNull,
-            table.storage.locationUri.map(_.toString).orNull, table.viewText.orNull))
+            table.storage.locationUri.map(_.toString).orNull, table.viewText.orNull,
+              upgrade_to match {case Some(target) => 1 case None => 0}, upgrade_to match {case Some(target) => target case None => ""}))
```

Comment:
remove the upgrade status column, we can infer it from `upgrade_target`. Please answer what would happen if the table is removed from the target catalog.


also add the same parsing to `_crawl` method in the `TablesCrawler` python's implementation, so that we can properly run the integration tests.